components/raftstore/src/store/snap.rs

```diff
@@ -1051,7 +1084,10 @@ impl Snapshot {
         // set snapshot meta data
         snap_data.set_file_size(total_size);
         snap_data.set_version(SNAPSHOT_VERSION);
-        snap_data.set_meta(self.meta_file.meta.as_ref().unwrap().clone());
+        let meta = self.meta_file.meta.as_mut().unwrap();
+        meta.set_start(start.into_inner());
+        meta.set_generate_duration_sec(t.saturating_elapsed().as_secs());
```

Comment:
can we not use `meta` to pass the context?

Reply:
I think it's too waster and complex  become more if we pass this stats by snapshot manager.